### PR TITLE
#1483 - Implemented About US Page (Responsive Desktop+Mobile)

### LINKED ISSUE #1483: [Desktop][Mobile] : Update the About Us page.
As a user,
When I click o the About Us button on the main men bar,
Then I should be able to see all the information about FightPandemics organization along with 2 CTAs: "Get Involved" and "Go to Help Board" buttons.

- When I click on the "Get Involved" button,
Then a popup should appear with 3 buttons: "Join as Volunteer", "Join as Ambassador" and "Join Student Program"
(These there buttons should redirect the user on the following links which should open on a new page: 
**"Join as Volunteer":** https://apply.workable.com/fightpandemics,  
**"Join as Ambassador":** https://apply.workable.com/fightpandemics/j/46D6EF3B44/, 
 **"Join Student Program":**  https://apply.workable.com/fightpandemics/j/58B157AAB2/)

- When I click on the "Go to Help Board" button,
Then I should be redirected to the Help Board.

Design Desktop: https://zpl.io/29W9pjl
Design Desktop with popup: https://zpl.io/25WmL9n
Design Mobile: https://zpl.io/aROKRRN
Design Mobile with popup: https://zpl.io/2GZ5L09

**Note:** 
1. All the content on the page should me same as the designs linked above. 
2. All the email addresses on the page should be clickable and when clicked should prompt to a email application.
3. All the social media logos are clickable and when clicked should redirect to that particular social media page on a new tab. Below are the links:

- LinkedIn: https://www.linkedin.com/company/fightpandemics

- Facebook: https://www.facebook.com/FightPandemics

- Twitter: https://twitter.com/fightpandemics

- Instagram: https://www.instagram.com/fightpandemics

4. All the supporter logos are clickable and when clicked should redirect to that particular page on a new tab. Below are the links:
**Life Time supporters:**
_Algolia: https://www.algolia.com
Hackolade: https://hackolade.com
Lokalise: https://lokalise.com
Miro: http://www.miro.com
SauceLabs: http://saucelabs.com
Sentry: https://sentry.io/welcome
Social Bee: https://socialbee.io
White Source: https://www.whitesourcesoftware.com_
**Current Supporters:**
_Airtable: https://airtable.com
AWS: http://aws.amazon.com
Auth0: http://auth0.com
Figma: https://www.figma.com
Gitkraken: https://www.gitkraken.com
Google Maps: https://maps.google.com
Lambda Test: https://www.lambdatest.com
MongoDb: https://www.mongodb.com
Notion: https://www.notion.so
PagerDuty: https://www.pagerduty.com
Prowly: https://prowly.com/en
Sendgrid: http://sendgrid.com
Seo4Ajax: https://www.seo4ajax.com
Slack: https://slack.com
Supermetrics: http://supermetrics.com
Twilio: http://www.twilio.com
Typeform: http://www.typeform.com
Zeplin: http://zeplin.io
Zendesk: http://www.zendesk.com
Workable: https://www.workable.com_
**Past Supporters:**
_Leypal: https://www.leypal.com
Calendly: https://calendly.com_

### PR DESCRIPTION
**Pull request by wadehrarshpreet**
### About US page [new Design]

https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5f4d44779a1b160f379bb2b0
https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5f4860489fc4fe35254bf3e1

This PR resolves #1483

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right-hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

This PR is continuation of 
https://github.com/FightPandemics/FightPandemics/pull/1532

### CODE REVIEW

**Comment by rabbi08** [comment]
@wadehrarshpreet @tejasm2 
While reviewing the "About Us" page on mobile (iPhone 11 pro max) I have following observations :

1. There is lots of white space observed which looks weird, great if we review the design and reduce this "blank space" user experience. 
Refer below screenshot 

https://share.getcloudapp.com/2Nuj26YX

https://share.getcloudapp.com/v1uxAEoE

https://share.getcloudapp.com/RBuWPB5D

https://share.getcloudapp.com/yAuloAXW

2. The page is loaded very slow as compared to other pages and due that the image containers are empty white space.

Please note as of now I have not tested the desktop version.

File: client/src/constants/theme.js [hunk 1]
```diff
@@ -72,6 +74,8 @@ const theme = {
     black: BLACK,
     white: WHITE,
     offWhite: OFF_WHITE,
+    gray:GRAY,
+    lighterBlack:LIGHTER_BLACK,
```
Comment (idjevm) [inline client/src/constants/theme.js]: Format seems a bit off here. Please reformat.

File: client/src/pages/AboutUs.js [hunk 2]
```diff
@@ -45,51 +48,102 @@ import typeformLogo from "assets/supporters-logos/typeform-logo.png";
...
+import lokaliseLogo from "assets/supporters-logos/lokaliseLogos.png";
+import saucelabsLogos from "assets/supporters-logos/saucelabsLogo.png";
+import whitesourceLogo from "assets/supporters-logos/whitesourceLogo.png";
+import lambdatestLogo from "assets/supporters-logos/lambdatestLogo.png";
+import prowlyLogo from "assets/supporters-logos/prowlyLogo.png";
+import seo4ajaxLogo from "assets/supporters-logos/seo4ajaxLogo.png";
+import supermetricsLogo from "assets/supporters-logos/supermetricsLogo.png";
+import figmaLogo from "assets/supporters-logos/figmaLogo.png";
+import googleMapsLogo from "assets/supporters-logos/googleMapsLogo.png";
+import awsLogo from "assets/supporters-logos/awsLogo.png";
```
Comment (idjevm) [inline client/src/pages/AboutUs.js]: Let's keep the imports organized in alphabetic order.

File: client/src/pages/AboutUs.js [hunk 3]
```diff
@@ -45,51 +48,102 @@ import typeformLogo from "assets/supporters-logos/typeform-logo.png";
...
+  [awsLogo, "http://aws.amazon.com"],
+  [aut0Logo, "http://auth0.com"],
+  [figmaLogo, "https://www.figma.com"],
+  [gitkrakenLogo, "https://www.gitkraken.com"],
+  [googleMapsLogo, "https://maps.google.com"],
+  [lambdatestLogo, "https://www.lambdatest.com"],
+  [mongodbLogo, "https://www.mongodb.com"],
+  [notionLogo, "https://www.notion.so"],
+  [pagerdutyLogo, "https://www.pagerduty.com"],
+  [prowlyLogo, "https://prowly.com/en"],
```
Comment (idjevm) [inline client/src/pages/AboutUs.js]: Same as above, let's organize them alphabetically.

File: client/src/pages/EditAccount.js [hunk 4]
```diff
@@ -138,7 +138,7 @@ function EditAccount(props) {
         );
       }
     })();
-  }, [userProfileDispatch]);
+  }, [t, userProfileDispatch]);
```
Comment (idjevm) [inline client/src/pages/EditAccount.js:141]: could we give a more descriptive name here for `t`? Could we change it to something else? Something more descriptive.

File: client/src/pages/EditOrganisationProfile.js [hunk 5]
```diff
@@ -179,7 +179,7 @@ function EditOrganisationProfile(props) {
         );
       }
     })();
-  }, [orgProfileDispatch, organisationId]);
+  }, [orgProfileDispatch, organisationId, t]);
```
Comment (idjevm) [inline client/src/pages/EditOrganisationProfile.js:182]: Let's change all the `t`s to something more descriptive.

File: client/src/pages/EditProfile.js [hunk 6]
```diff
@@ -167,7 +167,7 @@ function EditProfile(props) {
         );
       }
     })();
-  }, [userProfileDispatch]);
+  }, [t, userProfileDispatch]);
```
Comment (idjevm) [inline client/src/pages/EditProfile.js:170]: Same comment as above.

File: client/yarn.lock [hunk 7]
```diff
@@ -0,0 +1,14361 @@
...
+  integrity sha512-YKgNbG2dlzqMhA9NtI3/pbY16m3Yl/EeWBRa+lB1X1YaYxHrxNexiQYCLTWO/uDvAjLFMEDU+zR901waBtMtjQ==
+  dependencies:
+    tinycolor2 "^1.4.1"
+
+"@ant-design/colors@^4.0.5":
+  version "4.0.5"
+  resolved "https://registry.yarnpkg.com/@ant-design/colors/-/colors-4.0.5.tgz#d7d100d7545cca8f624954604a6892fc48ba5aae"
+  integrity sha512-3mnuX2prnWOWvpFTS2WH2LoouWlOgtnIpc6IarWN6GOzzLF8dW/U8UctuvIPhoboETehZfJ61XP+CGakBEPJ3Q==
+  dependencies:
+    tinycolor2 "^1.4.1"
```
Comment (idjevm) [inline client/yarn.lock]: Remove this fila completely please, unless you added a new library.  -> this is mentioned here -> https://github.com/FightPandemics/FightPandemics/wiki/How-to-create-a-Pull-Request

**Review by idjevm — CHANGES_REQUESTED** [review]
Great job @wadehrarshpreet ! Thank yo so much for this and also to @sujatha-m !


This are some thigns that quickly noticed while taking a look at the code, I have not look at the live page yet but will do soon and add comments accordingly

File: client/src/pages/EditAccount.js (hunk 4 above)
Comment (wadehrarshpreet) [reply client/src/pages/EditAccount.js:141]: I didn't made those changes when i merge staging, then eslint added this function as we are using it inside useEffect. `t` function is for the translation function provided by i18n.

**Review by jessica-lau — CHANGES_REQUESTED** [review]
@wadehrarshpreet Tested it on desktop and found the following:

1. There's a lot of white space on the page that should be reduced.
2. The bottom of this image is cut off. 

image

Tested it on iPhone X and Google Pixel 3 and there is a lot of white space showing on both.

**Comment by tejasm2** [comment]
Hey @rabbi08 , @jessica-lau ...regarding the white space, its actually as per the design from Zeplin, but dont know how it is looking on your screens, but we can surely reduce it a bit.

**Review by galeza — CHANGES_REQUESTED** [review]
LGTM
Only one question: First image should be cut?
Screen Shot 2020-10-05 at 11 12 55

**Comment by galeza** [comment]
Hi,
I tested on desktop and mobile Android 10.
Simulator:
iPhone 11 pro max:
Screen Shot 2020-10-05 at 11 16 39
iPhone XS:
Screen Shot 2020-10-05 at 11 19 09

**Comment by tejasm2** [comment]
> LGTM
> Only one question: First image should be cut?
> Screen Shot 2020-10-05 at 11 12 55

No it shoudnt be cut @galeza , its strange as on my desktop it is not cut. @wadehrarshpreet do you know why is this happening?

**Comment by jessica-lau** [comment]
> > LGTM
> > Only one question: First image should be cut?
> > Screen Shot 2020-10-05 at 11 12 55
> 
> No it shoudnt be cut @galeza , its strange as on my desktop it is not cut. @wadehrarshpreet do you know why is this happening?

I'm seeing it as cut on my desktop as well.

**Review by rabbi08 — CHANGES_REQUESTED** [review]
@wadehrarshpreet 
Still the image on mobile is cut, it displays only from 'Government --> Non profits'
But on the desktop it display 'Government -->Corporations'

Refer attached screenshot.
Mobile VS Desktop

**Comment by tejasm2** [comment]
> @wadehrarshpreet
> Still the image on mobile is cut, it displays only from 'Government --> Non profits'
> But on the desktop it display 'Government -->Corporations'
> 
> Refer attached screenshot.
> Mobile VS Desktop

Hey @rabbi08 , I think so cut is an issue........but displaying of governments through non profits for mobile is as per the designs on zeplin: https://zpl.io/b6rwLod . @wadehrarshpreet can you please check why the image is being cut?

**Comment by kchan335** [comment]
@tejasm2 It looks good, and I only found some inconsistent font size issues. Can we make the font size more consistent as following?

**Desktop** 
H1: Poppins/ bold/ 38px
H2: Poppins/ bold/ 32px
H3: Poppins/ Semibold/ 24px
P: work sans/ regular/ 22px

**Mobile**
H1: Poppins/ bold/ 26px
H2: Poppins/ bold/ 22px
H3: Poppins/ Semibold/ 20px
P: work sans/ regular/ 18px

**Comment by jessica-lau** [comment]
I'm seeing the image display fully on desktop now, but I'm still seeing the image cut on mobile.

image
image

**Review by jessica-lau — CHANGES_REQUESTED** [review]
As mentioned earlier, I'm seeing the image display fully on desktop now, but still seeing the image cut on mobile.

**Comment by wadehrarshpreet** [comment]
> As mentioned earlier, I'm seeing the image display fully on desktop now, but still seeing the image cut on mobile.

@jessica-lau Its as per design https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5f4860489fc4fe35254bf3e1

**Comment by rabbi08** [comment]
Agree @wadehrarshpreet it is as per zeplin, but @tejasm2 Please confirm if on mobile we should display the cut version of the image of FP-About US? Ideally should not. 

image

**Comment by tejasm2** [comment]
Yes @rabbi08 , maybe @kchan335 or @CamilaArias can suggest on the mobile part.

**Comment by jyothybalusu** [comment]
I have observed the following discrepancies on the web version on chrome.

1. COVID-info is missing on the top tight shortcuts.
Is there no particular order of how these shortcuts re displayed? The order of these 5 shortcuts on zeplin doesn’t match on the screen. Zeplin shows About us as the third on the top right where as the built in screen shows About us as the first option
2. The text By helping communities, In Zeplin the word By starts with a capital ‘B’ where as the built in page displays a small letter ’b’
3. Most of the places on the screen where long text is displayed on the About us Page the indentation on zeplin doesn’t exactly match on the built-in screen. Will this change according to the screen size based on the device it is opened?
4. There is a discrepancy in the screen design next to the corporation box in Zeplin and the screen. 
5. The video is not displayed in the part our story
6. Lifetime supporters and current supporters indentation differs to that of zeplin
7. Zendesk supporter is missing in the current supporters
8. Past supporters list order doesn’t match that of Zeplin
9. Follow us on Social media and its icons don’t match the indentation compared with zeplin
10. 
Screenshot 2020-10-20 at 13 09 13
Screenshot 2020-10-20 at 13 09 33

In the footer Get Involved option is missing and in addition it shows FAQ and Blog options in addition to that of Zeplin screen


Screenshot 2020-10-20 at 12 51 38
Screenshot 2020-10-20 at 12 51 58

**Comment by kchan335** [comment]
> Yes @rabbi08 , maybe @kchan335 or @CamilaArias can suggest on the mobile part.

I think it's fine to display the cut version on mobile because the image will look too small when displaying with full image.

**Comment by kchan335** [comment]
> 1. COVID-info is missing on the top tight shortcuts.

----> It's ok to get rid of COVID-info, if we don't have it for now.  

>    Is there no particular order of how these shortcuts re displayed? The order of these 5 shortcuts on zeplin doesn’t match on the screen. Zeplin shows About us as the third on the top right where as the built in screen shows About us as the first option

---->Help Board should be the first item if it is the first thing we'd like to show to our users....

**Comment by kchan335** [comment]
Is it possible to make Sign in as a button like Join Now on the navbar? They should look like a group.
Screen Shot 2020-10-21 at 1 59 47 AM

**Comment by tejasm2** [comment]
> I have observed the following discrepancies on the web version on chrome.
> 
> 1. COVID-info is missing on the top tight shortcuts.
>    Is there no particular order of how these shortcuts re displayed? The order of these 5 shortcuts on zeplin doesn’t match on the screen. Zeplin shows About us as the third on the top right where as the built in screen shows About us as the first option
> 2. The text By helping communities, In Zeplin the word By starts with a capital ‘B’ where as the built in page displays a small letter ’b’
> 3. Most of the places on the screen where long text is displayed on the About us Page the indentation on zeplin doesn’t exactly match on the built-in screen. Will this change according to the screen size based on the device it is opened?
> 4. There is a discrepancy in the screen design next to the corporation box in Zeplin and the screen.
> 5. The video is not displayed in the part our story
> 6. Lifetime supporters and current supporters indentation differs to that of zeplin
> 7. Zendesk supporter is missing in the current supporters
> 8. Past supporters list order doesn’t match that of Zeplin
> 9. Follow us on Social media and its icons don’t match the indentation compared with zeplin
> 
> Screenshot 2020-10-20 at 13 09 13
> 
> Screenshot 2020-10-20 at 13 09 33
> 
> In the footer Get Involved option is missing and in addition it shows FAQ and Blog options in addition to that of Zeplin screen
> 
> Screenshot 2020-10-20 at 12 51 38
> 
> Screenshot 2020-10-20 at 12 51 58

@jyothybalusu 
1. Its just in the design, but not yet completely developed, so we dont need it for now.
2. The "b" should be small actually as its a continuous sentence, zeplin design shows "B" mistakenly.
3. Yes, it will change a bit based on the screen size.
4. Can u explain more on the discrepancy?
5. Yes currently we dont have the video ready, we will add it once its ready later.
6. This too would change a bit based on the screen sizes.
7.  Good callout, we need to add Zendesk logo in current supporters @wadehrarshpreet 
8. I think this should be fine, the order wouldn't matter. 
9. This one needs to be changed @wadehrarshpreet

**Comment by wadehrarshpreet** [comment]
> > I have observed the following discrepancies on the web version on chrome.
> > 
> > 1. COVID-info is missing on the top tight shortcuts.
> >    Is there no particular order of how these shortcuts re displayed? The order of these 5 shortcuts on zeplin doesn’t match on the screen. Zeplin shows About us as the third on the top right where as the built in screen shows About us as the first option
> > 2. The text By helping communities, In Zeplin the word By starts with a capital ‘B’ where as the built in page displays a small letter ’b’
> > 3. Most of the places on the screen where long text is displayed on the About us Page the indentation on zeplin doesn’t exactly match on the built-in screen. Will this change according to the screen size based on the device it is opened?
> > 4. There is a discrepancy in the screen design next to the corporation box in Zeplin and the screen.
> > 5. The video is not displayed in the part our story
> > 6. Lifetime supporters and current supporters indentation differs to that of zeplin
> > 7. Zendesk supporter is missing in the current supporters
> > 8. Past supporters list order doesn’t match that of Zeplin
> > 9. Follow us on Social media and its icons don’t match the indentation compared with zeplin
> > 
> > Screenshot 2020-10-20 at 13 09 13
> > Screenshot 2020-10-20 at 13 09 33
> > In the footer Get Involved option is missing and in addition it shows FAQ and Blog options in addition to that of Zeplin screen
> > Screenshot 2020-10-20 at 12 51 38
> > Screenshot 2020-10-20 at 12 51 58
> 
> @jyothybalusu
> 
> 1. Its just in the design, but not yet completely developed, so we dont need it for now.
> 2. The "b" should be small actually as its a continuous sentence, zeplin design shows "B" mistakenly.
> 3. Yes, it will change a bit based on the screen size.
> 4. Can u explain more on the discrepancy?
> 5. Yes currently we dont have the video ready, we will add it once its ready later.
> 6. This too would change a bit based on the screen sizes.
> 7. Good callout, we need to add Zendesk logo in current supporters @wadehrarshpreet
> 8. I think this should be fine, the order wouldn't matter.
> 9. This one needs to be changed @wadehrarshpreet

@tejasm2 fixed requested changes.

**Comment by jessica-lau** [comment]
hey @wadehrarshpreet I have one more question. When viewing the page on mobile, the list of supporters are slightly off the grid leaning towards the right side. Is this something that can be fixed to make them more centered? The heading above the social media icons isn't fully centered as well. Thanks.

image

image

image

**Comment by kchan335** [comment]
Hi, we have new supporters and will need help to add their logos on our About Us page. CC @tejasm2 @jessica-lau @wadehrarshpreet 

Here is the link to the latest About Us design with new supporters: https://www.figma.com/file/EMKcg0eTnOlr7bRZGFdnZu/Web-app-product-spec?node-id=1%3A2

I've made all logos with the **same dimensions**, hoping it's helpful to make logos aligned. 

Here is the package of the lifetime supporters' logos:
[Updated lifetime supporters' logos.zip](https://github.com/FightPandemics/FightPandemics/files/5479731/Updated.lifetime.supporters.logos.zip)

The current supporters' logos:
[Updated current supporters' logos.zip](https://github.com/FightPandemics/FightPandemics/files/5479734/Updated.current.supporters.logos.zip)

The past supporters:
[Updated past supporters' logos.zip](https://github.com/FightPandemics/FightPandemics/files/5479736/Updated.past.supporters.logos.zip)

Mobile_lifetime supporters:
[Mobile_lifetime supporter's logos.zip](https://github.com/FightPandemics/FightPandemics/files/5479755/Mobile_lifetime.supporter.s.logos.zip)

Mobile_current supporters:
[Mobile_current supporters' logos.zip](https://github.com/FightPandemics/FightPandemics/files/5479756/Mobile_current.supporters.logos.zip)

Mobile_past supporters:
[Mobile_past supporters' logos.zip](https://github.com/FightPandemics/FightPandemics/files/5479758/Mobile_past.supporters.logos.zip)

**Comment by jessica-lau** [comment]
@wadehrarshpreet @kchan335 thanks for fixing the alignment of the logos in the grid to make them centered. Looks good now.

**Review by jessica-lau — APPROVED** [review]

**Review by galeza — APPROVED** [review]

File: client/src/pages/EditOrganisationProfile.js (hunk 5 above)
Comment (joshmorel) [reply client/src/pages/EditOrganisationProfile.js:182]: For `const { t } = useTranslation();` I think just the `t`  okay.

**Comment by Zaydme** [comment]
I have noticed that some of the assets are huge, this could drastically increase the loading time, and could make the page slow/unresponsive on old hardware and uncapable mobile devices.
image
There are some ways to optimize those gifs (jifs btw :joy:)

For example, I was able to reduce the size of this gif from **7.15MB** down to **0.97MB**
and here are the results 

# Original: 7.15MB (1200x1200)


# Optimized: 3.74MB (1200x1200)
using Lossy GIF compression (level: 50/200)


# Optimized: 1.96MB (1200x1200)
From 300 frames to 150 frames.


# Optimized: 0.97MB (1200x1200)
From 300 frames to 75 frames, but faster speed


For this optmization I used [ezgif](https://ezgif.com/)

**Comment by rabbi08** [comment]
@tejasm2 @kchan335 
- Under lifetime supporters we are having "nubela" but in the list of the supporters shared above, this supporter name does NOT exist. 

- Also we are not having a video as shown in the figma design shared above.

Please confirm that above updates are expected or need to be changed.

Other than that for testing this looks good on both Desktop and mobile. Please see below GIF.

image

**Comment by galeza** [comment]
> @tejasm2 @kchan335
> 
> * Under lifetime supporters we are having "nubela" but in the list of the supporters shared above, this supporter name does NOT exist.
> * Also we are not having a video as shown in the figma design shared above.
> 
> Please confirm that above updates are expected or need to be changed.
> 
> Other than that for testing this looks good on both Desktop and mobile. Please see below GIF.
> 
> image

@rabbi08 
Here is Slack conversation about Nubela: https://fightpandemics.slack.com/archives/C010P7A17V3/p1604413970190700?thread_ts=1604413607.190400&cid=C010P7A17V3
Maybe this info can help.

**Comment by tejasm2** [comment]
Yes, Nebula has been recently added to the supporters.

**Comment by kchan335** [comment]
@rabbi08 It seems like we don't have a video yet. And can we update the picture for We connect people? I noticed that the current picture is missing something... CC @tejasm2 

The updated picture is here:
[Mobile phone for We connect people.svg.zip](https://github.com/FightPandemics/FightPandemics/files/5502554/Mobile.phone.for.We.connect.people.svg.zip)

The rest looks good to me! Thanks a lot!

**Comment by kchan335** [comment]
@rabbi08 It seems like we don't have a video yet. And can we update the picture for We connect people? I noticed that the current pic is missing something. 

Here is the updated picture:
[Pic for We connect people.svg.zip](https://github.com/FightPandemics/FightPandemics/files/5502679/Pic.for.We.connect.people.svg.zip)

The rest looks good to me. Thanks a lot!

**Review by Zaydme — APPROVED** [review]
LGTM, great job @wadehrarshpreet!

**Review by idjevm — APPROVED** [review]
LGTM, are we working on the translations for other languages for this page as well? I noticed that some of them do not have its corresponding translation... @doheekim93 ?

**Comment by galeza** [comment]
> LGTM, are we working on the translations for other languages for this page as well? I noticed that some of them do not have its corresponding translation... @doheekim93 ?

@idjevm 
First we need to merge this PR and then new keys will be added to Lokalise. So we will work on translations and minor issues after merging this PR.

**Pull request merged**